zookeeper-api/src/test/java/org/apache/helix/zookeeper/impl/client/TestRawZkClient.java

```diff
@@ -1063,23 +1063,26 @@ public void testAsyncWriteByExpectedSession() throws Exception {
    * Tests getChildren() when there are an excessive number of children and connection loss happens,
    * the operation should terminate and exit retry loop.
    */
-  @Test(timeOut = 30 * 1000L)
+  @Test
   public void testGetChildrenOnLargeNumChildren() throws Exception {
     final String methodName = TestHelper.getTestMethodName();
     System.out.println("Start test: " + methodName);
+    String[] nodePaths = new String[110001];
     // Create 110K children to make packet length of children exceed 4 MB
     // and cause connection loss for getChildren() operation
     String path = "/" + methodName;
+    nodePaths[110000] = path;
 
     _zkClient.createPersistent(path);
 
     for (int i = 0; i < 110; i++) {
       List<Op> ops = new ArrayList<>(1000);
       for (int j = 0; j < 1000; j++) {
         String childPath = path + "/" + UUID.randomUUID().toString();
+        nodePaths[1000 * i + j] = childPath;
         // Create ephemeral nodes so closing zkClient deletes them for cleanup
         ops.add(
-            Op.create(childPath, new byte[0], ZooDefs.Ids.OPEN_ACL_UNSAFE, CreateMode.EPHEMERAL));
+            Op.create(childPath, new byte[0], ZooDefs.Ids.OPEN_ACL_UNSAFE, CreateMode.PERSISTENT));
       }
       // Reduce total creation time by batch creating znodes
       _zkClient.multi(ops);
```

Comment:
Forgive me for I might not have a good understanding on this but what is `ops` and what is the purpose of calling `multi` on it? I believe `multi()` isn't supported at the moment, just wondering if I am missing something. Thanks!

Reply:
Thanks for your comment!
`ops` is an iterable of `op` (which is a single operation https://zookeeper.apache.org/doc/r3.5.5/api/org/apache/zookeeper/Op.html). Transaction support is a zk server side feature so we already have it.
` multi() isn't supported at the moment` this means we don't have the multi() API in Helix ZKClient.
Please let me know if it makes sense.

Reply:
Got it, makes sense. Thanks for the clarification!